sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -1,5 +1,6 @@
 package io.sentry.android.core;
 
+import static io.sentry.MeasurementUnit.Duration.NANOSECOND;
```

Comment:
`m`: I feel like backend expects it to be in milliseconds, or did you align with MDX and iOS on that already?